lib/config/rules/stylistic-issues.js

```diff
@@ -1,8 +1,12 @@
 // see http://eslint.org/docs/rules/#stylistic-issues
 
 module.exports = {
+  // enforce linebreaks after opening and before closing array brackets
+  'array-bracket-newline': 'off',
```

Comment:
Disabled this, I misunderstood what the `multiline` option mean. 

> Perhaps multiline should be renamed to multiline-element to avoid confusion. 
https://github.com/eslint/eslint/issues/8981#issuecomment-317109119

Reply:
Boo, what a terrible option :/